tests/manifests/special-cases/no-strategy/kustomization.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
-  - ../../default
+  - ../../e2e
```

Comment:
If these manifest are based on `e2e` then how about we nest the `special-cases` directory under the `e2e` directory?

Reply:
and maybe we add a `default` directory under `e2e` which is where that manifest should go.

Come to think of it, but at this point we should probably make the `default` case "progressive" (and "pause and drain" can become a special case).